pandas/tests/series/methods/test_replace.py

```diff
@@ -254,9 +254,9 @@ def test_replace2(self):
         assert (ser[6:10] == -1).all()
         assert (ser[20:30] == -1).all()
 
-    def test_replace_with_dictlike_and_string_dtype(self):
+    def test_replace_with_dictlike_and_string_dtype(self, nullable_string_dtype):
```

Comment:
`arrow_string` behaves the same as `string`

```
>>> pd.__version__
'1.3.0.dev0+1219.gccc6a707e2'
>>> 
>>> s = pd.Series(["one", "two", np.nan], dtype="string")
>>> 
>>> s
0     one
1     two
2    <NA>
dtype: string
>>> 
>>> s.replace({"one": "1", "two": "2"})
0       1
1       2
2    <NA>
dtype: object
>>> 
```

is this correct?

Reply:
I would say that's a bug, but that's in any case unrelated for this PR (there are a few open issues related to `replace` with nullable dtypes, so it might already be covered)

Reply:
See eg https://github.com/pandas-dev/pandas/issues/40732 (for other nullable dtypes)